src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -32,50 +35,64 @@
-				.map( filter -> SearchFilterFiles.class.getResource( filter ) ) //
+				.map( SearchFilterFiles.class::getResource ) //
+				.filter( Objects::nonNull ) //
+				.map( URL::toExternalForm ) //
+				.map( URI::create ) //
+				.map( SearchFilterFiles::loadFilterFromUri ) //
 				.filter( Objects::nonNull ) //
-				.map( resource -> Paths.get( URI.create( resource.toString() ) ) ) //
-				.collect( Collectors.toList() ); //
+				.collect( Collectors.toList() );
+	}
+
+	private static Pair<Path, FilterLoader> loadFilterFromUri( final URI uri ) {
+		try {
+			final Path path = Paths.get( uri );
+			return Pair.of( path, FilterLoader.load( path ) );
+		} catch ( final FileSystemNotFoundException e ) {
+			try ( final FileSystem fs = FileSystems.newFileSystem( uri, Collections.emptyMap() ) ) {
+				final Path path = fs.provider().getPath( uri );
+				return Pair.of( path, FilterLoader.provide( path ) );
+			} catch ( final IOException ex ) {
+				log.error( "Could not load Filter at '{}'", uri, ex );
+				return null;
+			}
+		}
 	}
 
-	public static List<Path> getProjectFilterFiles() {
+	public static List<Pair<Path, FilterLoader>> getProjectFilterFiles() {
 		final Path resolveFilterPath =
 				ProjectConfiguration.getInstance().findProjectConfigFolder().resolve( FILTER_FOLDER );
 		if ( !resolveFilterPath.toFile().exists() ) {
 			return Collections.emptyList();
 		}
-		try ( Stream<Path> paths = Files.walk( resolveFilterPath ) ) {
+		try ( final Stream<Path> paths = Files.walk( resolveFilterPath ) ) {
 			return paths.filter( Files::isRegularFile ) //
 					.filter( file -> file.toString().endsWith( FILTER_EXTENSION ) ) //
+					.map( path -> Pair.of( path, FilterLoader.load( path ) ) ) //
 					.collect( Collectors.toList() ); //
 		} catch ( final IOException e ) {
 			log.error( "Exception accessing user filter folder '{}'.", resolveFilterPath, e );
 			return Collections.emptyList();
 		}
 	}
 
-	public static Map<Path, Filter> toPathFilterMapping( final List<Path> paths ) {
+	public static Map<Path, Filter> toPathFilterMapping( final List<Pair<Path, FilterLoader>> paths ) {
 		return paths.stream() //
-				.collect( Collectors.toMap( //
-						Function.identity(), //
-						SearchFilterFiles::toFilter ) );
-	}
-
-	private static Filter toFilter( final Path filter ) {
-		try ( final Stream<String> filterFileLines = Files.lines( filter ) ) {
-			return Loaders.load( filterFileLines ) //
-					.filter( Filter.class::isInstance ) //
-					.map( Filter.class::cast )//
-					.collect( Collectors.collectingAndThen( Collectors.toList(), CompoundFilter::new ) );
-		} catch ( final IOException e ) {
-			throw new UncheckedIOException( "Could not load '" + filter + "' file.", e );
-		}
+				.collect( Collectors.toMap( Pair::getLeft, pair -> {
+					final FilterLoader loader = pair.getRight();
+					try {
+						return loader.load();
+					} catch ( final IOException e ) {
+						log.error( "Could not load Filter for '{}'.", pair.getLeft() );
```

Comment:
I would suggest to ignore the actual exception:

```suggestion
						log.error( "Could not load Filter for '{}'.", pair.getLeft(), e );
```